ballerina/build.gradle

```diff
@@ -948,6 +947,7 @@ task testExamples() {
         'rabbitmq-transaction-producer',
         'rabbitmq-transaction-consumer',
         'jdbc-complex-type-queries', // Consists date time in output
+        'log-api',
```

Comment:
Any reason this can't be verified? This BBE is printing to the console right?

Reply:
That is because the output consists of the timestamp and that varies.

Reply:
Ack